packages/neo-one-smart-contract-lib/package.json

```diff
@@ -35,6 +35,7 @@
   "noCompile": true,
   "dependencies": {
     "@neo-one/smart-contract": "^1.0.0-alpha.7",
+    "@neo-one/smart-contract-compiler": "^1.0.0-alpha.19",
     "@neo-one/smart-contract-test": "^1.0.0-alpha.20"
```

Comment:
Remove all dependencies in all package.json that refer to `"@neo-one/smart-contract-test"`